test/abbreviate.js

```diff
@@ -25,6 +25,9 @@ test("abbreviate", assert => {
   assert.equal("1mm",      abbreviate(1000000, "es-ES"), "spanish locale");
   assert.equal("1,23t",    abbreviate(1234567890000, "es-ES"), "trillion in spanish");
 
```

Comment:
can you add some tests to test the decimal locale replacements? (these tests only check for thousands replacement)